sublime_linter.py

```diff
@@ -400,3 +412,20 @@ def remember_runtime(bid):
 
     with global_lock:
         elapsed_runtimes.append(runtime)
+
+
+def none_for_none(fn):
+    try:
+        return fn()
+    except Exception as exc:
+        if 'NoneType' in str(exc):
+            return None
+        else:
+            raise
+
+
+@lru_cache()
```

Comment:
I'm always scared when I see one of those. I'm scared of caching in general. Is this one safe?

Reply:
This is so that we don't spam users with the logging. We show each unique message only once in the status bar. (cache size by default is 128 items/messages). 👍 ?

Reply:
👍 👍